playbooks/roles/mongo_3_0/tasks/main.yml

```diff
@@ -322,7 +322,6 @@
 - name: create mongodb users in a replica set
   mongodb_user:
     database: "{{ item.database }}"
-    login_database: 'admin'
```

Comment:
That's good to hear. It didn't work for me lats time i tested and I spent a fair amount of time on it. The upstream folks were not surprised that it didn't work out of the box so I gave up  after a while. And yes arbiter config isn't supported that's why I proposed just using 3 data mongo nodes rather than 2 + arbiter. I don't really see the value in the arbiter node.